### DISCUSSION TITLE
Grasscutter | v1.6.1

### DISCUSSION THREAD

**Discussion by KingRainbow44** [Melledy/Grasscutter#2183] [Announcements]
# Changelog
- Fixed quests not completing when it had an enter region trigger
- Fixed statues not unlocking when talking to them
- Added missing argument for quest-related Lua function calls
- Added chest data from client for chests created through the quest entity packet
- Moved trial avatars having their alternate costume to a config option, disabled by default

## What's Changed
* Fix issues with regions by @scooterboo in https://github.com/Grasscutters/Grasscutter/pull/2179
* Remove mention of unstable branch in README.md by @DiXiaoO in https://github.com/Grasscutters/Grasscutter/pull/2181
* Fix a typo I left in Player.java by @scooterboo in https://github.com/Grasscutters/Grasscutter/pull/2180

## New Contributors
* @DiXiaoO made their first contribution in https://github.com/Grasscutters/Grasscutter/pull/2181

**Full Changelog**: https://github.com/Grasscutters/Grasscutter/compare/v1.6.0...v1.6.1

<hr /><em>This discussion was created from the release <a href='https://github.com/Grasscutters/Grasscutter/releases/tag/v1.6.1'>Grasscutter | v1.6.1</a>.</em>

**Comment by earthjasonlin** [comment]
现在所有的任务都可以正常做吗
